suggestedimages/locales/locale.py

```diff
@@ -21,6 +20,7 @@ def __init__(self, wikt=None):
         try:
             self.module = importlib.import_module(f'suggestedimages.locales.{wikt}')
         except ImportError:
+            print(f"Couldn't load module: {wikt}")
```

Comment:
Why did you leave this here? Yes, it's clear why but since we got some new instructions on how to do these reviews... :P Maybe consider importing and using logging instead.

Reply:
You're right it should be logged instead.